piccolo_admin/endpoints.py

```diff
@@ -558,6 +577,32 @@ def get_table_list(self, request: Request) -> t.List[str]:
         """
         return [i.table_class._meta.tablename for i in self.table_configs]
 
+    ###########################################################################
+
+    def get_translation(
+        self, request: Request, language: str
+    ) -> t.Union[JSONResponse, t.Dict[str, str]]:
+        """
+        Return a single language.
+        """
+        try:
+            language = request.path_params.get("language", None)
+            return [
+                i for i in TRANSLATIONS if i.get("language", None) == language
+            ][0]
+        except IndexError:
+            return JSONResponse(
+                {"error": "Language does not exist."}, status_code=422
+            )
+
+    def get_translations_list(
+        self, request: Request
+    ) -> t.Dict[str, t.Dict[str, str]]:
+        """
+        Returns a list of all languages.
+        """
+        return {i["language"]: i for i in TRANSLATIONS}
```

Comment:
If `TRANSLATIONS` was a dict, we could do `list(TRANSLATIONS.keys())`.

Reply:
With dictionary it will be as simple as `return TRANSLATIONS`.